src/app/store/slices/storage/storage.thunks/downloadItemsThunk.ts

```diff
@@ -32,27 +34,37 @@ export const downloadItemsThunk = createAsyncThunk<void, DriveItemData[], { stat
 
     // * 1. Creates tasks
     for (const item of items) {
-      if (item.isFolder) {
-        const taskId = tasksService.create<DownloadFolderTask>({
-          action: TaskType.DownloadFolder,
-          relatedTaskId: taskGroupId,
-          folder: item,
-          compressionFormat: 'zip',
-          showNotification: true,
-          cancellable: true,
-        });
+      if (item?.taskId === undefined) {
```

Comment:
If `item?.taskId === undefined` means that is a download (which is a bit weird, btw), could be more understandable to use a variable for this:

`const taskIsADownload = item?.taskId === undefined;`

and then:

`if (taskIsADownload) {...}`

Reply:
I added it at first because I thought the taskId was a number to avoid the problem of false true by negating a possible 0. Then I saw that the taskId is a string and I forgot to change this. 
I add as you say in a boolean variable that identifies the reason for the check because if not it is very likely that if you don't have context you won't understand the reason for the check.